pkg/tfvars/libvirt/cache.go

```diff
@@ -28,7 +28,7 @@ func (libvirt *Libvirt) UseCachedImage() (err error) {
 		return nil
 	}
 
-	logrus.Infof("Fetching OS image: %s", filepath.Base(libvirt.Image))
+	logrus.Infof("Fetching qemu URL: %s", filepath.Base(libvirt.Image))
```

Comment:
nit: The log message should describe the rrsource being fetched, but "URL" is just *how* we're fetching it.  If you don't like "OS image", how about "QEMU image"?  Or "QCOW2" (although I don't know if we actually require this to be QCOW2 or if libvirt can handle other formats too)?